homeassistant/components/wallbox/sensor.py

```diff
@@ -167,8 +167,12 @@ def __init__(
 
     @property
     def native_value(self) -> StateType:
```

Comment:
It looks like this only returns a subset of `StateType`

In a future PR, please type this to what is actually returns.